src/faction-boards/nevlas.ts

```diff
@@ -15,10 +15,10 @@ const nevlas: FactionBoardRaw = {
   },
   income: ["2k,4o,15c,q,up-sci", "+o,k"],
   handlers: {
-    [`build-${Building.PlanetaryInstitute}`]: (player: Player) => player.data.tokenModifier = 2,
     freeActionChoice:  (player: Player, pool: any[]) => {
       pool.push(...freeActionsNevlas);
       if (player.data.hasPlanetaryInstitute()) {
+        pool.length = 0;
```

Comment:
Nice. I didn't know arrays could be resized this way!

Reply:
Actually, that was the only way to let it work. Everything else wasn't resetting ```pool``` in the emit event :)

Reply:
Probably `pool.splice(0)` would have worked. But your way is nicer :)